csuntechlab/iSTART
# STR-228 -- FE: Fix Pie Chart on Mobile.

# Description
The legend text of the pie chart will not be cut off and this achieved by shortening the `boxWidth` and on smaller screens(iPhone 5) the word Percentage was shorten to Pct..
  
# Testing
1. Go to browser responsive tool and select regular smartphone size
2. Go to a module with a pie chart.
3. Text should not be cut off.
4. Repeat the same process but for responsive tool select iPhone 5.
6. Do note the Percentage text is determined on load and not as the browser window is rigorously changing.
